src/basic_motion_references.cpp

```diff
@@ -82,7 +82,7 @@ BasicMotionReferenceHandler::~BasicMotionReferenceHandler() {
   }
 };
 
-bool BasicMotionReferenceHandler::sendCommand() {
+bool BasicMotionReferenceHandler::checkMode() {
   // TODO: Check comparation
   // if (this->current_mode_ != desired_control_mode_)
   if (this->current_mode_.yaw_mode != desired_control_mode_.yaw_mode ||
```

Comment:
This comparison is made 3 times if we send pose+speed+thrust, maybe we should try to optimize this @RPS98